From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <rafal@milecki.pl>
Subject: [PATCH] ARM: dts: BCM5301X: Add serial= to the bootargs
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It's enough to have proper stdout-path for getting serial working but
for some reason LEDE doesn't offer "Please press Enter to activate this
console." unless ttyS0 is specified.

This is a workaround to get serial working in LEDE.

Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
---

--- a/arch/arm/boot/dts/bcm47081-tplink-archer-c5-v2.dts
+++ b/arch/arm/boot/dts/bcm47081-tplink-archer-c5-v2.dts
@@ -13,7 +13,7 @@
 	model = "TP-LINK Archer C5 V2";
 
 	chosen {
-		bootargs = "earlycon";
+		bootargs = "console=ttyS0,115200 earlycon";
 	};
 
 	memory {
